LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2003-2007 The Kepler Project, 2013-2022 Matthew Wild
```

Comment:
Date-bumping the license is fine, but lets do it in a different commit and maybe even PR, it is a housekeeping thing and the git blame situation gets really noisy if important commits that might have unintended consequences also touch a bunch of unrelated bits.